irohad/multi_sig_transactions/impl/hash.cpp

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "multi_sig_transactions/hash.hpp"
+
+#include <functional>
+#include <string>
+
+#include "cryptography/public_key.hpp"
+#include "interfaces/common_objects/peer.hpp"
+#include "interfaces/iroha_internal/transaction_batch.hpp"
+
+namespace iroha {
+  namespace model {
```

Comment:
strange namespace

Reply:
Well, that is what was in the header file.